proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -39,6 +39,26 @@ service ResolveService {
   rpc GetRemotePackageVersion(GetRemotePackageVersionRequest) returns (GetRemotePackageVersionResponse) {
```

Comment:
Why does this still exist?

Reply:
Was going to delete in another PR but I'll add it into this one